core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -773,6 +773,28 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(true)
           .build();
+  // Client local cache metrics
+  public static final MetricKey CLIENT_CACHE_BYTES_READ_CACHE =
+      new Builder(Name.CLIENT_CACHE_BYTES_READ_CACHE)
+          .setDescription("Total number of bytes read from the Local Cache.")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey CLIENT_CACHE_BYTES_READ_EXTERNAL =
+      new Builder(Name.CLIENT_CACHE_BYTES_READ_EXTERNAL)
+          .setDescription("Total number of bytes read from external storage due to a cache miss "
+              + "on the local cache.")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey CLIENT_CACHE_BYTES_REQUESTED_EXTERNAL =
+      new Builder(Name.CLIENT_CACHE_BYTES_REQUESTED_EXTERNAL)
+          .setDescription("Total number of bytes the user requested to read which resulted in a "
+              + "cache miss. This number may be smaller than CacheBytesReadExternal due to chunk "
```

Comment:
what is `CacheBytesReadExternal`? Should we reference the PK variable instead?

Reply:
Updated: #10907